docs/rules/extensions.md

```diff
@@ -6,10 +6,12 @@ In order to provide a consistent use of file extensions across your code base, t
 
 ## Rule Details
 
-This rule has one option which could be either a string or an object. If it is `"never"` (the default value) the rule forbids the use for any extension. If `"always"` then the rule enforces the use of extensions for all import statements.
+This rule either takes one string option, one object option, or a string and an object option. If it is the string `"never"` (the default value) the rule forbids the use for any extension. If the string `"always"` then the rule enforces the use of extensions for all import statements.
```

Comment:
If it is the string `"never"` (the default value)
-->
`If it is the string `"never"` (the default value), then
(for consistency)

If the string `"always"` then the rule
-->
If it is the string `"always"` then the rule